### DISCUSSION TITLE
Finding the mayaUsdProxyShape node from a certain prim?

### DISCUSSION THREAD

**Discussion by subing85** [Autodesk/maya-usd#3821] [Q&A]
How to get mayaUsdProxyShape from prim?
this is example code

```
import mayaUsd

usdProxyShape = "|stage|mayaUsdProxyShape"

stage = mayaUsd.ufe.getStage(usdProxyShape)
prim = stage.GetPrimAtPath('/hello/world')

usdProxyShape = ?
```

If someone is able to assist, it should be appreciable.
Subin

**Comment by seando-adsk** [comment]
From the prim you can get the stage and then use the python binded function `stagePath` which returns a string.

```
stage = mayaUsd.ufe.stagePath(prim.GetStage())
print(stage)
# |stage1|mayaUsdProxyShape
```

Then using that you can create a ufe scene item or maya object.

Sean

**Reply by Subin-Jackfruit** [reply]
Thank you very much master.
